tests/llmobs/conftest.py

```diff
@@ -60,6 +60,16 @@ def mock_llmobs_eval_metric_writer():
     patcher.stop()
 
 
+@pytest.fixture
+def mock_llmobs_evaluator_runner():
```

Comment:
not a huge fan of the mocked tests. Would definitely be ideal if we could generate evals and assert that the data is emitted when `flush` is called. But since we have this pattern already I'm ok with it.

Reply:
Going to add this an item under [refactor ragas]